notebooks/MAST/plot_exoplanet_lightcurves/plot_exoplanet_lightcurves.ipynb

```diff
@@ -0,0 +1,1678 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #7.               "User-agent":"python-requests/"+version}
</pre><p>In our testing, this line seems unnecessary; you don't have to specify the python version. Try removing this line, and if your code still functions, we should take this out to simplify the header and the required imports</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ScienceBetter/notebooks-stsci/pull/19/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>